pymeasure/instruments/attocube/anc300.py

```diff
@@ -408,9 +408,9 @@ def read(self):
                              f"command with message {msg}")
         return self._extract_value(msg)
 
-    def wait_for(self, query_delay=0):
+    def wait_for(self, query_delay=None):
         """Wait for some time. Used by 'ask' to wait before reading.
 
-        :param query_delay: Delay between writing and reading in seconds.
+        :param query_delay: Delay between writing and reading in seconds. None is default delay.
```

Comment:
Maybe indicate what happens if no delay is provided, i.e., that the `self.query_delay` value will be used?

Reply:
I expanded th comments.